crates/stdarch-verify/tests/x86-intel.rs

```diff
@@ -711,10 +713,10 @@ fn equate(
     match (t, &intel[..]) {
         (&Type::PrimFloat(32), "float") => {}
         (&Type::PrimFloat(64), "double") => {}
-        (&Type::PrimSigned(8), "__int8" | "char") => {}
+        (&Type::PrimSigned(8), "__int8" | "constexpr int" | "char") => {}
         (&Type::PrimSigned(16), "__int16" | "short") => {}
         (&Type::PrimSigned(32), "__int32" | "const int" | "int") => {}
-        (&Type::PrimSigned(64), "__int64" | "long long") => {}
+        (&Type::PrimSigned(64), "__int64" | "long long" | "size_t") => {}
```

Comment:
`size_t` should be an unsigned type, preferably `usize` (you can define the LLVM intrinsic signature with `stride` as `usize`, LLVM doesn't care about the signedness or type of integer arguments, just their bit width)

Reply:
Added `usize` as a primitive in x86-intel, please check if that what you expect